### PR TITLE
Add tooltips to discussion post buttons

### LINKED ISSUE #9588: Add a tooltip to discussion post buttons
as someone who rarely uses the discussion its very unintiuitve that these posts are "post" buttons, I always think they simply select the post type.

grafik

### PR DESCRIPTION
**Pull request by RockRoller01**
someone please double check that all the titles make sense, not too familiar with the discussion system

closes #9588

### CODE REVIEW

File: resources/lang/en/beatmaps.php [hunk 1]
```diff
@@ -67,15 +67,42 @@
         ],
 
         'message_type' => [
-            'disqualify' => 'Disqualify',
```
Comment (nanaya) [inline resources/lang/en/beatmaps.php:70]: this should be done in a way that doesn't remove all existing translations. Add a suffix, use different group, etc.

**Review by nanaya — CHANGES_REQUESTED** [review]

File: resources/lang/en/beatmaps.php (hunk 1 above)
Comment (RockRoller01) [reply resources/lang/en/beatmaps.php]: resolved by using a different group

File: resources/js/beatmap-discussions/new-discussion.tsx [hunk 2]
```diff
@@ -455,6 +455,7 @@ export class NewDiscussion extends React.Component<Props> {
         props={{
           'data-type': type,
           onClick: this.post,
+          title: trans(`beatmaps.discussions.message_type_title.${typeText}`),
```
Comment (nanaya) [inline resources/js/beatmap-discussions/new-discussion.tsx:458]: title tooltip behaviour is rather bugged in chrome when the button is disabled. Wrap it with a div or something and put the title there (it's still kind of buggy but at least it's less bad).

**Review by nanaya — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
Another one which can probably be review-applied from our end just to clean up open PRs.

**Review by nanaya — APPROVED** [review]

**Pull request merged**